configure.ac

```diff
@@ -37,6 +37,12 @@ AC_CONFIG_HEADERS([config.h])
 AC_DEFINE([CONFIG_H_INCLUDED], [], [defined if config.h included])
 PKG_PROG_PKG_CONFIG
 
+# set configure arguments
+AC_CONFIGURE_ARGS=`echo "${ac_configure_args}" | sed "s/ //"`
```

Comment:
That variable isn't populated on my system. It's empty. For now, I use `sed` directly.